### ISSUE TITLE
Syntax error, unrecognized expression: #collapseFeature@canva/web0

### ISSUE THREAD

**Issue by joscha** [gkushang/cucumber-html-reporter#92]
I seem to get an error when running this on a json report from a project that is scoped, e.g.:

```
  "name": "@canva/web",
```
in `package.json`, produces the following HTML:

```html
            <div id="collapseFeature@canva/web0" class="panel-collapse collapse">
                <div class="panel-body">
                    
                    
                    <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                
                                <div class="tags"> </div>
                                <a data-toggle="collapse" href="#collapseScenario@canva/web0_0">
                                    <div>

                                        <div style="padding-right: 30px">
                                            <i class="glyphicon glyphicon-chevron-right"></i>
                                            <i class="glyphicon glyphicon-chevron-down"></i>
                                            <b>Scenario:</b>Editor header has a design title
                                        </div>
                                        <div>
                            <span class="label-container">
```

which then in turn produces the following error:

![cucumber_feature_report_and_cucumber_report_html](https://user-images.githubusercontent.com/188038/28011787-ebff0678-65a6-11e7-81b2-69da3c4d29cc.png)

**Comment by gkushang** [comment]
fix published with v2.0.2 👍

**Comment by joscha** [comment]
Thank you!
